### PR TITLE
Feature/display map on search

### PR DESCRIPTION
**Pull request by KShih**
Nicely display the search results with the map.

### CODE REVIEW

**Comment by ab7289** [comment]
While this looks good, the intent wasn't to get rid of the list view that we had in the search beforehand. Muhammad is still working on updating the styling on that, and I wasn't planning on getting rid of it.  I was thinking more along the lines of we would keep the search page as it is, but there would be a button or something that said "Display on Map" that would take you to the map view.  

This way people could either scroll through the list to view the choices, or see them geographically on the map.

**Comment by KShih** [comment]
We are not getting rid of list of the results. The map is displaying at the left of the results.

**Comment by ab7289** [comment]
Yes this affectively would be getting rid of the list view. Please review the screen caps from Muhammad's PR.  The updated styling would not fit nicely on the page with the map, and I would like the user to have the option to either view locations as a list of cards as in Muhammad's PR or on your map.

**Comment by KShih** [comment]
@ab7289 Changed.

File: search/templates/search/search.html [hunk 1]
```diff
@@ -1,15 +1,62 @@
...
     $(document).ready(function () {
         $("#complaints_button").click(function () {
             $("#complaints").toggle();
         });
     });
+
+    var markerList = [];
+    var mapCenter = [0,0];
+
+    {% for item in search_data.locations %}
```
Comment (alldne) [inline search/templates/search/search.html]: I think this `{% for item in search_data.locations %}` would create redundant javascript code to our result html file.

How about this? (I didn't run this code. Please take the brief idea from it)
```
let markerList = [];

const locations = [
{% for item in search_data.locations %}
[ {{ item.latitude }}, {{ item.longitude}}, "{{ item.google_map_url }}" ],
{% endfor %}
];

let sumLatitude = 0;
let sumLongitude = 0;

for (const loc of locations) {
  markerList.push(loc);
  sumLatitude += loc[0];
  sumLongitude += loc[1];
}

const mapCenter = [ sumLatitude / markerList.length, sumLongitude / makerList.length ];
```

Comment (KShih) [reply search/templates/search/search.html]: @alldne Good catch! Updated.

**Comment by ab7289** [comment]
It also looks like the button still displays, even if there aren't any search results. Also when you click on the map button when there aren't any search results it doesn't appear to load completely.

Screen Shot 2019-11-25 at 6 10 18 PM
Screen Shot 2019-11-25 at 6 10 09 PM

**Comment by ab7289** [comment]
other than that tho I think this looks really nice!

**Comment by KShih** [comment]
@ab7289 Oh! Good catch and it should be fixed. After testing your PR will do it right away.

**Pull request merged**